tests/lodash-extras-tests.js

```diff
@@ -0,0 +1,256 @@
+import { expect } from 'chai';
```

Comment:
I noticed throughout this file that you are randomly missing semi-colons. Just FYI.